src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int WORKING_DATE_NUMBER = 0;
+    private static final int NAME_NUMBER = 1;
+    private static final int WORK_HOURS_PER_DAY_NUMBER = 2;
+    private static final int PRICE_FOR_ONE_WORK_HOUR = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder("Report for period ");
+        stringBuilder.append(dateFrom).append(" - ").append(dateTo);
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
checklist issue

Reply:
Sorry, i don't understand what you mean as this error